JASP-Engine/JASP/R/bainttestbayesianindependentsamples.R

```diff
@@ -238,9 +238,8 @@ BainTTestBayesianIndependentSamples <- function(jaspResults, dataset, options, .
 
 	  descriptivesTable$addColumnInfo(name="v",                    title = "", type="string")
 		descriptivesTable$addColumnInfo(name="group",                title = "Group", type="string")
-	  descriptivesTable$addColumnInfo(name="N",                    title = "N", type="integer")
+	  descriptivesTable$addColumnInfo(name="N",                    title = "n", type="integer")
 	  descriptivesTable$addColumnInfo(name="mean",                 title = "Mean", type="number")
-	  descriptivesTable$addColumnInfo(name="sd",                   title = "Std. Deviation", type="number")
```

Comment:
Any reason in particular why we're dropping the standard deviation?

Reply:
Yeah, it is probably not what people are interested in when calling bain. All we do is compare means, so then mean + se.mean + conf.int is important. Also, for coefficients it would not be standard to report standard deviations, standard practice would be the standard error. I would say, for standard deviations, visit descriptive statistics. Bain would then just be for inference. What do you think?

Reply:
I don't think it hurts anyone by being there, but I have no strong opinion either way, so it's up to you

Reply:
Alright, leaving it out then. If Herbert really wants it back, I'll put it back :p